templates/docs/examples/patterns/contextual-menu/_script.js

```diff
@@ -77,6 +77,22 @@ function setupAllContextualMenus(contextualMenuToggleSelector) {
       }
     }
   });
+
+  // Add handler for closing menus when the focus leaves
+  var dropdowns = document.querySelectorAll('.p-contextual-menu__dropdown');
```

Comment:
Feels like this could be a part of `setupAllContextualMenus` function rather than separate loop over all contextual menus.